### ISSUE TITLE
A11y_Visual Studio Code Client_Code Editor_Screen Reader: JAWS/NVDA is not announcing curly braces ({}) in the code editor when we are using the down arrow key.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#204693]
[([Check out Accessibility Insights!](https://accessibilityinsights.io/))]- Identify accessibility bugs before check-in and make bug fixing faster and easier.

### GitHubTags:
#A11yTCS; #A11ySev2; #DesktopApp; #Win32; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #WCAG1.3.1; #ScreenReader; #JAWS; #NVDA; #A11yMAS; 

### Environment and OS details:
Version: 1.87.0-insider (user setup)
Microsoft Windows 11 Enterprise 22H2 Build 22621.3007
JAWS Version: 2023.22.12.23ILM.
NVDA Version: 2022.2.3

### Repro Steps:
1. Open Visual Studio Code Insiders editor.
2. Tab till code panel.
3. Write curly braces {} in the code.
5. Verify whether the JAWS/NVDA is not announcing curly braces ({}) in the code editor when we are using the down arrow key or not.

### Actual Result:
JAWS/NVDA is not announcing curly braces ({}) in the code editor when we are using the down arrow key.

### Expected Result:
JAWS/NVDA should announce the curly braces ({}) in the code editor when we are using the down arrow key.

### User Impact:
Screen reader only users will not be able to get any information about the curly braces present in the code editor.

### Attachment:
ScreenReader

JAWS:

**Comment by kupatkar99** [comment]
@meganrogge verified the issue in below environment:
Visual Studio code Client
Version: 1.88.0-insider (user setup)
issue is still repro'ing with both the screen reader. with JAWS virtual PC mode it is announcing and with NVDA up and down arrow key it is not being announced by screen reader. Please find added attachment for reference.

https://github.com/microsoft/vscode/assets/98530069/93f8268c-4af4-4511-a6e3-c5c5670780d2


https://github.com/microsoft/vscode/assets/98530069/6c2d28fd-600b-4503-b933-de246a294932

**Comment by AnKushSingh05** [comment]
IRT Bug: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/219938
